plugins/woocommerce/src/Stats/McStats.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare( strict_types = 1 );
+
+/**
+ * WooCommerce MC Stats package
+ */
+
+namespace Automattic\WooCommerce\Stats;
+
+use Automattic\Jetpack\A8c_Mc_Stats;
```

Comment:
Does this work if Jetpack is not available on a site?

Reply:
Good question! This is using the [imported class](https://github.com/Automattic/jetpack-a8c-mc-stats/blob/6b7a74bd7c08773d3a156877cad20290ac87f7e4/src/class-a8c-mc-stats.php#L13) which is added in composer dependency [here](https://github.com/woocommerce/woocommerce/pull/49658/files#diff-62b5ff93172c7be053df8541c8702199d88e52766e3e7f3dc215dd5b6dfe22d4R22). Thus, this shouldn't rely on Jetpack plugin.

Reply:
Yes, like @ilyasfoo said, this should work without Jetpack plugin.

Reply:
Are we expecting any other variations of stat classes?? If not, I feel like we can just put the file in the `Internal` directory

Reply:
Not that I know of for now. I think that's a good idea. I'll move it there.

Reply:
Moved `McStats` to `./src/Internal` in 7346fbb1f66a4a15433412ba949643cc0b0afc7d